src/components/RightHandMenu/LinkToSource/LinkToSource.css

```diff
@@ -0,0 +1,10 @@
+.rt__link {
```

Comment:
Looks like on the trello ticket image, the link is underlined.

Reply:
Added!